docs/DeveloperGuide.md

```diff
@@ -221,7 +222,65 @@ for subsequent prompts.
 ---
 
 ### Parsing User Input and Getting the Right Command
-**_TODO_**: Explain how the app parses user input and determines which `Command` subclass object to instantiate.
+
+![Obtain and Parse User Input](uml/sequenceDiagrams/miscellaneous/images/obtainAndParseUserInput.png)
```

Comment:
Perhaps before the diagram is shown, can say something like "Below is a SD and explanation of how the ....... works"

Reply:
Thank you for the feedback. Personally, I think the additional statement prior to the diagram is redundant as my SD already has a title describing the SD.

Nevertheless, I will keep your suggestion in mind and if the rest of the team feels that an additional statement is indeed necessary, I will add it in. 🙂